app/assets/stylesheets/application.scss

```diff
@@ -17,12 +17,12 @@
 @import "global-styles/utilities";
 
 // file css rules that are targeted at specific components under components
+@import "components/AttachmentList";
 @import "components/ButtonGroupToggleButton";
 @import "components/search-modal";
 @import "components/search-results";
 
```

Comment:
General question: The files should be written in camelcase?

Reply:
I'm accustomed to give the scss files the exact same name as the corresponding component. But I'm not sure weather that's an obvious pattern to others. Especially in this project where naming has plenty of inconsistencies. What do you think?